wire-desktop-beta/.SRCINFO

```diff
@@ -1,34 +1,24 @@
 pkgbase = wire-desktop-beta
-	pkgdesc = Modern, private messenger. Based on Electron.
+	pkgdesc = End-to-end encrypted messenger with file sharing, voice calls and video conferences
 	pkgver = 3.5.2881
-	pkgrel = 1
+	pkgrel = 2
 	url = https://wire.com/
 	arch = x86_64
-	arch = i686
 	license = GPL3
 	makedepends = cargo
 	makedepends = npm
 	makedepends = python2
 	makedepends = git
 	makedepends = yarn
-	depends = alsa-lib
-	depends = gconf
-	depends = gtk2
-	depends = libxss
-	depends = libxtst
-	depends = nss
+	depends = electron
 	depends = xdg-utils
-	optdepends = hunspell-en: for English spellcheck support
-	optdepends = noto-fonts-emoji: for colorful emoji made by Google
-	optdepends = ttf-emojione: for colorful emoji made by EmojiOne
+	optdepends = hunspell-en_US: for English spellcheck support
+	optdepends = emoji-font: colorful emoji
 	provides = wire-desktop
-	conflicts = wire-desktop-bin
```

Comment:
`-bin` will probably go away after `wire-desktop` is in [community], and in any case I think it's `-bin` that should conflict with our packages, not the other way around